### PR TITLE
Add full custom MWI

### PR DESCRIPTION
**Pull request by delosh653**
Add full custom MWI functionality, with uploads directly in the app.

### CODE REVIEW

File: Data/Cleaned/HSE_MWI_ZCTA_Mental_Wellness_Index_Black.csv [hunk 1]
```diff
@@ -10943,7 +10943,7 @@
...
-"34108",78.3273755930284,97.0010721396845,81.6432385979989,93.5732846625008,98.8724246736373,93.2685763781476,97.9072573160923,96.672956753078,80.6854939039562,83.4512124711316,,80.8747978308439,17.5661605206074,74.527750292345,4.63822724622079,78.994401931077,52.0840012824623,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,75.4832474226804,21.4804375134042,28.7447532093508,94.1174667402367,27.9173149588798,93.3261772529985
+"34108",78.327543175725,97.0010721396845,81.6432385979989,93.5732846625008,98.8724246736373,93.2685763781476,97.9072573160923,96.672956753078,80.6854939039562,83.4512124711316,,80.8747978308439,17.5661605206074,74.527750292345,4.63822724622079,78.994401931077,52.0840012824623,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,75.4847815414826,21.4804375134042,28.7447532093508,94.1174667402367,27.9173149588798,93.3261772529985
```
Comment (karenmjiang) [inline Data/Cleaned/HSE_MWI_ZCTA_Mental_Wellness_Index_Black.csv:10946]: Just for record keeping, what led to the value changes for some of the data?

File: app.R [hunk 2]
```diff
@@ -931,18 +916,120 @@ ui <- fluidPage(
...
+          column(
+            width = 8,
+            HTML("<center><h2>Change weights to create your own Mental Wellness Index (MWI)!</h2></center>"),
+            HTML(paste0(
+              "<p align = 'justify'>",
+              "Weights used in the Mental Wellness Index control the relative influence each measure has on the total MWI; higher numbers inidcate a higher influence. If you adjust the weights to 0, a measure has no influence on the MWI.",
+              "<br><br>",
+              "To adjust the weights in the Mental Wellness Index, follow the instructions below. If you want to add your own data to the MWI, go to the \"Add Local Data to MWI\" section. Note that data uploaded to this application is not kept -- it is deleted once you leave the page, including any processing done to it.",
+              "<ol>",
+              "<li>Update weights for each measure in the table below as desired by doubleclicking the 'Updated Weights' column, then editing the measure to the desired amount (0 or a positive number).</li>",
```
Comment (karenmjiang) [inline app.R:935]: Maybe another note that the weights do not need to add to 100 (optional)

**Comment by karenmjiang** [comment]
After running the "Adjust MWI Weights" section, the success message persists on the "Add Local Data to MWI" and "Add Local Data to MWI on your Computer" pages.

File: app.R [hunk 3]
```diff
@@ -931,18 +916,120 @@ ui <- fluidPage(
...
+          column(width = 2)
+        )
+      ),
+      tabPanel(
+        title = div("Add Local Data to MWI", class = "explore"),
+        fluidRow(
+          column(width = 2),
+          column(
+            width = 8,
+            HTML("<center><h2>Add local data to create your own Mental Wellness Index (MWI)!</h2></center>"),
```
Comment (karenmjiang) [inline app.R:966]: I would change the headers here to match what is in the dropdown
"Add Local Data to MWI"

File: app.R [hunk 4]
```diff
@@ -931,18 +916,120 @@ ui <- fluidPage(
...
+          column(width = 2)
+        )
+      ),
+      tabPanel(
+        title = div("Add Local Data to MWI on Your Computer", class = "explore"),
+        fluidRow(
+          column(width = 2),
+          column(
+            width = 8,
+            HTML("<center><h2>Add local data to create your own Mental Wellness Index (MWI)!</h2></center>"),
```
Comment (karenmjiang) [inline app.R:1028]: Same here: "Add Local Data to MWI on Your Computer"

File: Data/Cleaned/HSE_MWI_ZCTA_Mental_Wellness_Index_Black.csv (hunk 1 above)
Comment (delosh653) [reply Data/Cleaned/HSE_MWI_ZCTA_Mental_Wellness_Index_Black.csv:10946]: I updated one of the scaling factors that was mistyped in Metadata.xlsx (100 should have been 1) and reran the pipeline -- while that doesn't really change any of the rankings in general (and shouldn't) , it looks like one of the rankings changed by numerical error, and doesn't change any interpretations.

File: app.R (hunk 4 above)
Comment (delosh653) [reply app.R]: done!

File: app.R (hunk 3 above)
Comment (delosh653) [reply app.R]: done!

File: app.R (hunk 2 above)
Comment (delosh653) [reply app.R]: done!

**Pull request merged**